sources/packages/web/nginx.conf

```diff
@@ -0,0 +1,30 @@
+user  nginx;
+worker_processes  1;
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+events {
+  worker_connections  1024;
+}
+http {
+  include       /etc/nginx/mime.types;
+  default_type  application/octet-stream;
+  log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                    '$status $body_bytes_sent "$http_referer" '
+                    '"$http_user_agent" "$http_x_forwarded_for"';
+  access_log  /var/log/nginx/access.log  main;
+  sendfile        on;
+  keepalive_timeout  65;
+  server {
+    listen       8080;
+    server_name  localhost;
```

Comment:
Should this not be updated to the real hostname?
I guess that, if present, nginx will try to validate the hostname.

Reply:
if we just change the configuration for server_name it won't block. Nginx uses the the first server block as the default server. However if we set a default server to return 404, then that will prevent the server to accept requests without the configured hosts.
```
  server {
    listen 8080 default_server;
    server_name _;
    # Configuration for the default server block
    return 404;
  }
  server {
    listen       8080;
    server_name  test.com;
    location / {
      root   /app;
      index  index.html;
      try_files $uri $uri/ /index.html;
    }
    error_page   500 502 503 504  /50x.html;
    location = /50x.html {
      root   /usr/share/nginx/html;
    }
  }
```

Should we configure it now in this ticket?

Reply:
This will help to understand how it works: http://nginx.org/en/docs/http/request_processing.html

Reply:
Thanks for the further explanation, I got it now.
For this PR I believe that we can remove the `server_name localhost` and add the improvement suggested above using the `server_name_;` in an upcoming ticket.